Filters/src/ParticleCodeFilter_module.cc

```diff
@@ -32,9 +32,11 @@ namespace mu2e {
 
       struct ParticleCodeSelector {
 	PDGCode::type pdgCode_;
-	ProcessCode creationCode_, terminationCode_;
+	int creationCode_, terminationCode_;
```

Comment:
PDGCode::type should be prefered to an int.    If a parameter is optional then std::optional<PDGCode::type> can be used here instead of relying on an expected range of "valid" integer values.

Reply:
Formerly these variables were ProcessCode, PdgCode would not parse.  I will make them optional.